www/roster/main.rb

```diff
@@ -167,6 +167,14 @@
   _html :podlings
 end
 
+# gives a list of podlings that would report in a given month based on current status
+get '/podlings/reporting/:month.json' do |month|
```

Comment:
Ok, I can see that as an issue, though its weird that I'm not getting prompted.

What do you think about exposing another one of these sinatra apps in a different directory for public info?